rtris.py

```diff
@@ -4,11 +4,176 @@
+      -V, --version   display version information and exit
+      -d, --debug     print debug information (default)
+      -D, --no-debug  don't print debug information
+
+    Exit Status:
+      (using CommonCodes v1.0.0
+       <https://mfederczuk.github.io/commoncodes/v/1.0.0.html>)
+
+Report bugs at: https://github.com/RiedleroD/Rtris/issues
+Rtris repository: https://github.com/RiedleroD/Rtris""" % (USAGE)
+VERSION_INFO="""\
+Rtris %s
+Copyright (c) %s %s
+License CC BY-SA 4.0: Creative Commons Attribution-ShareAlike 4.0 <http://creativecommons.org/licenses/by-sa/4.0>
+This is free software: you are free to change and redistribute it.
+There is NO WARRANTY, to the extent permitted by law.
+
+Written by %s.""" % (VERSION, COPYRIGHT_YEAR, COPYRIGHT_HOLDER, AUTHORS)
+
+def eprint(*args, **kwargs):
+	from sys import stderr
+	print(*args, file=stderr, **kwargs)
+
+def opt_help(opt, arg):
+	if arg != None:
+		eprint("%s: %s: too many arguments: 1" % (argv[0], opt))
+		exit(4)
+	print(HELP)
+	exit(0)
+def opt_version_info(opt, arg):
+	if arg != None:
+		eprint("%s: %s: too many arguments: 1" % (argv[0], opt))
+		exit(4)
+	print(VERSION_INFO)
+	exit(0)
+
+debug=True
+def dprint(*args, **kwargs):
+	global debug
+	if debug:
+		print(*args, **kwargs)
+def opt_debug(opt, arg):
+	if arg != None:
+		eprint("%s: %s: too many arguments: 1" % (argv[0], opt))
+		exit(4)
+	global debug
+	debug=True
+def opt_no_debug(opt, arg):
+	if arg != None:
+		eprint("%s: %s: too many arguments: 1" % (argv[0], opt))
+		exit(4)
+	global debug
+	debug=False
+
+# options are stored in this array here as tuples
+# tuple[0]: array of single character strings representing the short options
+# tuple[1]: array of strings representing the long options
+# tuple[2]: boolean value that decides whether or not the option requires an
+#            argument. if no argument is given, the handler will still be passed
+#            a None value, so the handler has to check for a missing argument
+# tuple[3]: the option handler function.
+#            the first parameter will be the name of the option used in the
+#            command line with the dashes. (e.g.: "-h", "--help", ...)
+#            the second parameter will be the argument given to the option on
+#            the command line. it doesn't matter what the value of tuple[2] is,
+#            this parameter can either be a string or None
+# tuple[4]: if True, the option is high priority and will be executed BEFORE
+#            checking for invalid options and BEFORE the low priority options.
+#            if False, the option is low priority and will be executed AFTER
+#            checking for invalid options and AFTER the high priority options
+options=[
+	(["h"], ["help"],     False, opt_help,         True),
+	(["V"], ["version"],  False, opt_version_info, True),
+	(["d"], ["debug"],    False, opt_debug,        False),
+	(["D"], ["no-debug"], False, opt_no_debug,     False)
+]
```

Comment:
instead of --debug and --no-debug there should be --debug=True|False or just --debug (since this is a front-end game, not a library, the arguments won't stack)